test-util/src/lib.rs

```diff
@@ -47,3 +50,24 @@ where
 
     runtime.block_on(func(cache_name, credential_provider))
 }
+
+pub fn create_doctest_client() -> (CacheClient, String) {
+    let cache_name =
+        env::var("TEST_CACHE_NAME").expect("environment variable TEST_CACHE_NAME should be set");
+
+    let credential_provider =
+        CredentialProviderBuilder::from_environment_variable("TEST_API_KEY".to_string())
```

Comment:
same comments here about whether it would be sane to use a default for test_cache_name if it's missing, and whether we should just use MOMENTO_API_KEY